samcli/lib/build/workflow_config.py

```diff
@@ -143,6 +144,7 @@ def get_workflow_config(
     selectors_by_build_method = {
         "makefile": BasicWorkflowSelector(PROVIDED_MAKE_CONFIG),
         "dotnet7": BasicWorkflowSelector(DOTNET_CLIPACKAGE_CONFIG),
+        "rustcargolambda": BasicWorkflowSelector(RUST_CARGO_LAMBDA_CONFIG),
```

Comment:
I wonder if we can simplify the `rustcargolambda` keyword 🤔

Reply:
How about just `cargolambda`? 
Because `cargolambda` is the underlying tool that performs the build, and `cargolambda` already implies it's for Rust (unless there's another language that also has a tool called Cargo-Lambda)